remote_config/tests/remote_config_test.cc

```diff
@@ -703,5 +703,7 @@ TEST_F(RemoteConfigTest, GetInfo) {
   EXPECT_EQ(info.last_fetch_status, kLastFetchStatusFailure);
   EXPECT_EQ(info.last_fetch_failure_reason, kFetchFailureReasonThrottled);
 }
+
```

Comment:
Probably just revert the change to this file.
I you look at the CMake file in this folder, this test appears to be disabled. When we moved to GitHub we don't have the same infra so a number of unit tests like this are no longer run and we rely more on the integration tests.

Reply:
reverted